examples/controlnet/.gitignore

```diff
@@ -0,0 +1 @@
+controlnet-model
```

Comment:
This we don't need no?

Reply:
It's the default directory we write checkpoints, etc.. into. I think it makes sense to ignore?

Reply:
I don't think people should run the script inside the their main github repo tbh, think the script should be linked from a Hub model repo and uploaded with it. E.g.:

```
git clone <your-hub-repo>
cp -r diffusers/examples/controlnet/train_control_net.py <your-hub-repo>
run script
```

But if you feel strongly ok for me

Reply:
Ah I always run from w/in source. I don't feel strongly can remove

Reply:
https://github.com/huggingface/diffusers/pull/2545/commits/cb49716906011baef85201a95ae27f56f7783043 done!